CheerpJ loader now is called directly from path stored in .env

Orpheus-3145 opened the pull request:
Changed made:

- .env file now has the path of the CheerpJ loader itself
- removed async call for reading the old path to fetch CheerpJ loader, using now dynamic JS to add the <script> (that contains the loader source) tag to <head>

alexp-sssup commented:
We should stick to 4.0 @Orpheus-3145 please use the public 4.0/loader.js path, not the internal 4.0_NNN one.

JavaFiddle is based on Java 8, which is now very stable.

Orpheus-3145 commented:
I updated the path with the 4.0 version

alexp-sssup commented on src/lib/CheerpJ.svelte:
```diff
@@ -34,17 +22,15 @@
-		<!-- CheerpJ is loading -->
-	{:then src}
-		{#if src}
-			<script {src} on:load={onLoad}></script>
```
Build the script using the svelte syntax, we can hardcode the known URL and remove the onMount handler

Orpheus-3145 replied:
If you are suggesting to do something like this:
```
<svelte:head>
    <script {src} on:load={onLoad}></script>
</svelte:head>
```
It will not work, that is way I used raw js instead.
I also saw some [posts](https://github.com/sveltejs/svelte/issues/8301) on github addressing the same problem; if we do not want to stick with ugly js, a possible solution would be to upgrade svelteKit to a newer version, but I'm not sure about that.

alexp-sssup replied:
You don't need onLoad at all. If the script is directly added to the page then you are guaranteed that by the time later scripts are executed the loader is ready to be used.

Orpheus-3145 commented:
Update:
1. removed file .env that contained the old path for CheerpJ loader
2. CheerpJ loader script is loaded hard-coded directly in app.html
3. removed manually adding <script> in the head using javascript, CheerpJ.svelte now just inits CheerpJ and renders its component

Orpheus-3145 commented on src/app.html:
```diff
@@ -10,6 +10,10 @@
 			src="https://plausible.leaningtech.com/js/script.js"
 		></script>
+		<script
+			defer
```
neither `async` nor `defer` were indeed necessary

Orpheus-3145 commented:
done the changes

alexp-sssup commented on src/app.html:
```diff
@@ -10,6 +10,9 @@
 		></script>
+		<script
+			src="https://cjrtnc.leaningtech.com/4.0/loader.js"
+		></script> 			<!-- scripts for CheerpJ loader -->
```
This comment is both redundant and poorly formatted, remove it

Orpheus-3145 replied:
gone
